main.cpp

```diff
@@ -65,10 +71,37 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int temperature = 20;
+    int phaseShift = -90;
+    int voltage = 15;
+
+    unsigned int age = 5;
+    unsigned int rms = 95;
+    unsigned int halfLife = 30;
+
+    char switch1State = 2;
+    char modeSelector = 9;
+    char midiChannel = 16; 
```

Comment:
use single quotes to define `char` types;
`char switch1State = '2';`  or `'a'`; 
- [ ] revise.